.github/workflows/lint-and-test.yaml

```diff
@@ -93,13 +93,6 @@ jobs:
           --config .github/ct.yaml \
           --helm-extra-args "--timeout 1m"
 
-      # TODO: Upgrade path.
-      # I need to comment the upghrade path until we have a first release.
-      # Comment for the future me:
-      #     This also failed because the port was used: set a strategy to recreate.
-      #- name: Test charts' upgrade path
-      #  run: ct --config .github/ct.yaml install --upgrade --all
-
```

Comment:
There is no way that we can test the upgrade path because this chart does not have version as it is created at release time.

Maybe, in a future, we might be able to test it but we should test it when we have the version and we manually add it to `Chart.yaml` using `yq` or something.

I will let this out of scope for this iteration.